packages/react/src/feature-app-loader.tsx

```diff
@@ -34,33 +31,34 @@ export class FeatureAppLoader extends React.PureComponent<
   FeatureAppLoaderProps,
   FeatureAppLoaderState
 > {
-  public readonly state: FeatureAppLoaderState;
+  public readonly state: FeatureAppLoaderState = {};
 
   private mounted = false;
   private loggedLoadingError = false;
 
-  private readonly asyncFeatureAppDefinition: AsyncValue<
-    FeatureAppDefinition<unknown>
-  >;
-
   public constructor(props: FeatureAppLoaderProps) {
     super(props);
 
-    const {manager, src} = props;
+    const {manager, src: browserSrc, nodeSrc} = props;
+    const src = inBrowser ? browserSrc : nodeSrc;
 
-    this.asyncFeatureAppDefinition = manager.getAsyncFeatureAppDefinition(src);
+    if (!src) {
+      return;
```

Comment:
This is impossible since only `nodeSrc` is optional.

(Just to make sure I haven't misunderstood your comment: You don't want to handle cases, that the compiler would also catch, do you?)

Reply:
yes that was not so obvious. nor does this GitHub UI yet offer any built-in compiler inference